EnglishFilter/sections/antiadblock.txt

```diff
@@ -452,6 +452,8 @@ falkirkherald.co.uk#@#.text_ads
 !#endif
 ! https://github.com/AdguardTeam/AdguardFilters/issues/41759
 @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=faucetcrypto.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/42081
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=hindustantimes.com
```

Comment:
I think that for apps and Chrome/Firefox/Opera extensions we can use `$empty` modifier in this case.
```
!+ PLATFORM(windows, mac, android, ext_chromium, ext_ff, ext_opera)
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$empty,important,domain=hindustantimes.com
!+ NOT_PLATFORM(windows, mac, android, ext_chromium, ext_ff, ext_opera)
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=hindustantimes.com
```